Example/nativeStack.js

```diff
@@ -51,33 +44,15 @@ export class Stack extends Component {
       stack: this.state.stack.filter(v => key !== v),
     });
   }
```

Comment:
Can you add blank lines between all class members? For all above methods too if you can (between `constructor` and `push` etc.)

```suggestion
  }
  
```